lib/SIL/Parser/SILParserTranslationUnitState.h

```diff
@@ -0,0 +1,62 @@
+//===--- SILParserState.h - SILParserTUState declaraion -------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2019 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef SWIFT_SIL_SILPARSERSTATE_H
+#define SWIFT_SIL_SILPARSERSTATE_H
+
+#include "swift/Basic/LLVM.h"
+#include "swift/Parse/ParseSILSupport.h"
+#include "swift/Parse/Parser.h"
+#include "swift/SIL/SILModule.h"
+
+//===----------------------------------------------------------------------===//
+// SILParserState
+//===----------------------------------------------------------------------===//
+
+namespace swift {
+
+// This has to be in the 'swift' namespace because it's forward-declared for
+// SILParserState.
+class SILParserState : public SILParserStateBase {
```

Comment:
Given the class has now been renamed to `SILParserState`, should the file be renamed back to `SILParserState.h`?

Reply:
I think that makes sense. I'll change it back.